app/javascript/modules/listingDetails/ListingDetailsPricingTable.tsx

```diff
@@ -213,7 +223,9 @@ const buildAccordions = (
                     <div>
                       {renderInlineMarkup(
                         t("listings.incomeRange.minMaxPerMonth", {
-                          min: Math.round(occupancy?.absoluteMinIncome).toLocaleString(),
+                          min: Math.round(
+                            forceZeroInRange ? 0 : occupancy?.absoluteMinIncome
+                          ).toLocaleString(),
```

Comment:
With the nested conditional, it's getting hard to see what the intended behavior is. Just want to double check that this is the the behavior for each scenario:
1. `occupancy?.absoluteMinIncome <= 0 && forceZeroInRange`
    - "$0 to $X per month"
2. `occupancy?.absoluteMinIncome <= 0 && !forceZeroInRange`
    - "up to $X per month"
3. `occupancy?.absoluteMinIncome > 0 && forceZeroInRange`
    - "$0 to $X per month"
4. `occupancy?.absoluteMinIncome > 0 && !forceZeroInRange`
    - "$Y to $X per month"

There's probably some way to express all the scenarios explicitly, while also keeping the code compact, but I haven't figured one out yet.

Reply:
Yes, that is correct